### PR TITLE
AQ messaging example update

### PR DESCRIPTION
**Pull request by danielkec**

### CODE REVIEW

File: examples/messaging/docker/oracle-aq/Dockerfile [hunk 1]
```diff
@@ -13,8 +13,12 @@
...
 #
+FROM container-registry.oracle.com/database/express:latest
 
-FROM oracle/database:18.4.0-xe as base
-
-
-COPY init.sql /docker-entrypoint-initdb.d/setup/
\ No newline at end of file
+ENV "ORACLE_PWsD"="frank"
+ENV "ORACLE_PWD"="frank"
```
Comment (senivam) [inline examples/messaging/docker/oracle-aq/Dockerfile:18]: remove line 18 and if possible change the password to 'changeit'

File: examples/messaging/oracle-aq-websocket-mp/src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (c) 2020, 2024 Oracle and/or its affiliates.
```
Comment (senivam) [inline examples/messaging/oracle-aq-websocket-mp/src/main/resources/application.yaml]: in new files it's enough to have one CP year

**Review by senivam — CHANGES_REQUESTED** [review]

File: examples/messaging/docker/oracle-aq/Dockerfile (hunk 1 above)
Comment (danielkec) [reply examples/messaging/docker/oracle-aq/Dockerfile:18]: It is used across messaging examples, such change should belong to standalone PR

**Review by senivam — DISMISSED** [review]
LGTM

**Review by senivam — APPROVED** [review]
LGTM

**Pull request merged**